lesson15/index.html

```diff
@@ -1 +1,31 @@
-
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1">
+   <title>lesson15</title>
+   <link rel="stylesheet" href="style.css">
+</head>
+<body id="body">
+   <div id="js-wrapper">
+      <input type="button" id="js-openModalBtn" value="Click Me!!" >
+      <ul id="js-ul">
+      </ul>
+      <div id="js-modal" class="modal">
+         <form method="" action="" class="form">
+            <div class="modal__inputID">
+               <p>ID</p>
```

Comment:
IMO
I think it's better to use <label> tags to implement this.
The label tag can be used to associate it with an input field.

Reply:
Thank you for your comments.
I changed that method to use label tags.

Reply:
The way it is written now, there is no association.

The following is a quote from MDN.
> <label> を <input> 要素に関連付けるには、 <input> に id 属性を設定しなければなりません。そして <label> に for 属性を設定して、値を input の id と同じにします。

Please refer to this document.
https://developer.mozilla.org/ja/docs/Web/HTML/Element/label

Reply:
I'm sorry. I forgot that.
I attached id to label tags. 
I apologize for the unnecessary hassle.
1e645aa

Reply:
This is a minor point, but I don't think` js- `is necessary, because it is not handled by JS.

Reply:
The input element is handled by JS, and I had to match its id attribute with the for attribute of the label element, so that's how I did it.
In this case, does it mean that the id attribute of the input element is not necessary to be prefixed with js-?

Reply:
I'm sorry...😭
You're right 🤝
I gave a wrong review, so please forget about it as soon as possible...

Reply:
Don't worry. 
I was relieved.
Thanks for taking time out of your busy schedule to comment again and again!